Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM node:carbon
+FROM node:carbon-slim
```

Comment:
@Zooloo2014 I am personally happy with using `slim` if alpine is too much trouble. `slim` is still very small compared to full versions

Reply:
:+1: for going with the slim version. Much cleaner solution